tests/unit/core/data/transform_libs/test_mmcv.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+"""Unit tests of mmcv data transform."""
+
+import numpy as np
+import pytest
+from mmcv.transforms.builder import TRANSFORMS
+from otx.core.config.data import SubsetConfig
+from otx.core.data.entity.base import ImageInfo, OTXDataEntity
+from otx.core.data.transform_libs.mmcv import LoadImageFromFile, MMCVTransformLib
+from otx.core.types.transformer_libs import TransformLibType
+
+
+class TestLoadImageFromFile:
+    def test_transform(self) -> None:
+        transform = LoadImageFromFile()
+        data_entity = OTXDataEntity(
+            np.ndarray((224, 224, 3)),
+            ImageInfo(0, (224, 224, 3), (224, 224, 3), (0, 0, 0), (1.0, 1.0)),
+        )
+        out = transform.transform(data_entity)
+        assert out["img_shape"] == (224, 224)
+        assert out["ori_shape"] == (224, 224)
+
+
+class TestMMCVTransformLib:
+    def test_get_builder(self) -> None:
+        assert MMCVTransformLib.get_builder() == TRANSFORMS
+
+    def test_check_mandatory_transform(self) -> None:
+        class MockNormalTransform:
+            def __call__(self, entity: OTXDataEntity) -> OTXDataEntity:
+                return entity
+
+        class MockMandatoryTransform:
+            def __call__(self, entity: OTXDataEntity) -> OTXDataEntity:
+                return entity
+
+        MMCVTransformLib._check_mandatory_transforms(
```

Comment:
I personally think that it is not likely to add a test for the private function unless it is really needed. You might can design a test `MMCVTransformLib.generate` (public function) to test this behavior.

Reply:
Done by [06bf724](https://github.com/openvinotoolkit/training_extensions/pull/2711/commits/06bf7249cc4c8d47b5bec215c1fb3ef8d9929022)